src/platform/imx8/include/platform/platform.h

```diff
@@ -15,6 +15,7 @@
 #include <arch/lib/wait.h>
 #include <sof/drivers/interrupt.h>
```

Comment:
@IulianOlaru249 Lets use 'platform: imx: Implement panic routine' for commit subject. 

Also, please always use oops instead of Oops. Starting with random capital letter in the body of the commit doesn't look good.